rdmo/management/assets/js/api/MultiSiteApi.js

```diff
@@ -0,0 +1,12 @@
+
+import BaseApi from 'rdmo/core/assets/js/api/BaseApi'
+
+class MultiSiteApi extends BaseApi {
+
+  static UpdateElementAction(element, action) {
```

Comment:
I think I would rather add 3 functions to the 3 API classes, makes it more readable.